Makefile

```diff
@@ -6,7 +6,7 @@ KERNELDIR?=/lib/modules/$(shell uname -r)/build
 DRIVERDIR?=$(shell pwd)/driver
 
 # Where kernel drivers are going to be installed
-MODULEDIR?=/lib/modules/$(shell uname -r)/kernel/drivers/hid
+MODULEDIR?=/lib/modules/$(shell uname -r)/extra
```

Comment:
Any explanation why?

Reply:
The 'kernel/drivers/hid' is used as part of kernel's source tree, but the openrazer driver is not in, so It better to move to 'extra'. 

But when the driver's code will be merged into kernel - the path should be fixed.

Reply:
I'd rather not change it as it's working nicely for multiple years :)

Reply:
Let's replace our pentium i-core to i80286 - it works for multiple years :-((
I see no reason to argue - my target is "push merge to master brunch for all mamba elite's users" and this is not the last legacy in the project, so let it be.